framework/src/base/FEProblem.C

```diff
@@ -1202,22 +1194,14 @@ FEProblem::addFunction(std::string type, const std::string & name, InputParamete
   for (THREAD_ID tid = 0; tid < libMesh::n_threads(); tid++)
   {
     MooseSharedPointer<Function> func = MooseSharedNamespace::static_pointer_cast<Function>(_factory.create(type, name, parameters, tid));
-    if (_functions[tid].find(name) != _functions[tid].end())
-      mooseError("Duplicate function name added to FEProblem: " << name);
```

Comment:
This error appears to be deleted. Is it not an issue anymore? I guess we didn't have a test anyway. What happens if you add two functions with the same name? You can't do that through the input file in any natural way but what if an Action does this? I seem to recall that this was added for that very case.